bin/rialto-runtime/Cargo.toml

```diff
@@ -55,6 +55,11 @@ version = "0.1.0"
 default-features = false
 path = "../../modules/currency-exchange"
 
+[dependencies.pallet-substrate-bridge]
+version = "0.1.0"
+default-features = false
```

Comment:
More a question - can there be some negative effects of not-enabling "std" feature when runtime is compiled with std? Probably something related to panic-handling or disabled logs/debug info, ...?

Reply:
I'm not entirely sure tbh. But I assume that having `no_std` code running in a `std` environment is more forgiving than the other way around